COMMERCE-9767 - It should not be possible to set Tier Price quantity as "0"

victorcdp opened the pull request:
The input for the tier price quantity had no restrictions, allowing for the user to input 0 or negative numbers.
Proposed fix is to add checks on both back and front end that prevent the input o 0 or negative numbers.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

chanmj96 commented:
ci:test:relevant

brianikim commented:
Cl

brianikim commented:
ci:stop

BarbaraCabrera committed:
LPS-166401 Adds ItemDescriptor and ItemSelectorViewDescriptor for AssetEntry

BarbaraCabrera committed:
LPS-166401 Use AssetBrowserDisplayContext to get the searchContainer to reuse the logic

BarbaraCabrera committed:
LPS-166401 Add assetEntryId and groupDescriptiveName to the returnType

BarbaraCabrera committed:
LPS-166401 Use the proper icon for each asset className

BarbaraCabrera committed:
LPS-166401 SF fix typo

lfbesada committed:
LPS-166401 Add the assetType to the returnType

lfbesada committed:
LPS-166401 Add status

lfbesada committed:
LPS-166401 SF extract to a field

Commit:
LPS-166000 funcio prototype (will be assemblage of all the "function" integrations)

Commit:
LPS-166000 [workflow] open up ScriptLanguage enum into interface so we can inject more options

brianchandotcom committed:
LPS-166000 Sort

brianchandotcom committed:
LPS-166000 Avoid chaining

brianchandotcom committed:
Revert "LPS-166000 funcio prototype (will be assemblage of all the "function" integrations)"

This reverts commit 3a43d7fb4a631d8468b7c1ecf891dab86c2dced9.

brianchandotcom committed:
LPS-166000 SF

brianchandotcom committed:
LPS-166000 Copy the pattern in FunctionObjectActionExecutorImplConfiguration. Type: git ls-files **Function**Configuration.java

Commit:
LPS-166240 Util-ify

Commit:
LPS-166240 apply usage

apply

brianchandotcom committed:
LPS-166240 SF

brianchandotcom committed:
LPS-166240 Wordsmith

brianchandotcom committed:
LPS-166403 Remove old class

brianchandotcom committed:
LPS-166403 Don't expose invocation handler

brianchandotcom committed:
LPS-166240 Move it to ConfigurableUtil. I'm not entirely happy with this, but I like it slightly more than osgi-util/FactoryUtil

brianchandotcom committed:
LPS-166404 Add remove method

brianchandotcom committed:
LPS-166405 Always use the full impl name #sfme

brianchandotcom committed:
LPS-166405 Rename #sfme

brianchandotcom committed:
LPS-166406 These classes are not OSGi components. They are instantiated by JSONWebServiceClientFactoryImpl and only used in DDMRESTDataProvider.java:

		JSONWebServiceClient jsonWebServiceClient =
			_jsonWebServiceClientFactory.getInstance(
				HashMapBuilder.<String, Object>put(
					"hostName", _getHostName(uri.getHost())
				).put(
					"hostPort", _getHostPort(uri.getPort(), uri.getScheme())
				).put(
					"keyStore", _getKeyStore()
				).put(
					"login", ddmRESTDataProviderSettings.username()
				).put(
					"password", ddmRESTDataProviderSettings.password()
				).put(
					"protocol", uri.getScheme()
				).put(
					"trustSelfSignedCertificates",
					_ddmDataProviderConfiguration.trustSelfSignedCertificates()
				).putAll(
					_getProxySettingsMap()
				).build(),
				false);

		String response = null;

		try {
			response = jsonWebServiceClient.doGet(
				absoluteURL, _getParametersArray(allParameters));
		}
		finally {
			jsonWebServiceClient.destroy();
		}

brianchandotcom committed:
LPS-166000 SF complains without this commit. I'm not sure if we need this line or if we should remove this line. See LPS-88783 for origin of the SF complaint.

brianchandotcom committed:
LPS-166000 Buy space

brianchandotcom committed:
LPS-166000 Sort

brianchandotcom committed:
LPS-166000 Rename

brianchandotcom committed:
LPS-166000 Wordsmith

brianchandotcom committed:
LPS-166000 Aiming to move this out after _authorize is implemented

brianchandotcom committed:
LPS-166240 Remove duplicate dependency

Commit:
COMMERCE-9767 - Adding checks to prevent price tier quantity from being less than 1

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
